aea/helpers/exec_timeout.py

```diff
@@ -150,6 +152,8 @@ def _set_timeout_watch(self) -> None:
 
         :return: None
         """
+        if os.name == "nt":
+            pytest.skip("signal.settimer non available on Windows.")
```

Comment:
Hmm surely we don't just want to skip during testing?

Reply:
yes my bad, will fix now